### PR TITLE
KVM Workload Cluster Release v15.0.0

### PR DESCRIPTION
**Pull request by tfussell**
For https://github.com/giantswarm/roadmap/issues/324

### CODE REVIEW

File: kvm/v15.0.0/release.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v15.0.0
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.7.1
+  - name: chart-operator
+    version: 2.18.0
```
Comment (rossf7) [inline kvm/v15.0.0/release.yaml:13]:
```suggestion
    version: 2.18.1
```

@yulianedyalkova Could you use this patch release?

Its for this PM https://github.com/giantswarm/giantswarm/issues/18324

It just increases the memory limit for chart-operator.  In the puma workload clusters the prometheus-operator app often hits the limit as its a large chart with a lot of values.

File: kvm/v15.0.0/release.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+# devctl release create --name v15.0.0 --provider kvm --base 14.1.2 --app cert-exporter@1.7.1 --app chart-operator@2.18.0 --app coredns@1.6.0@1.8.3 --app net-exporter@1.10.2 --component kubernetes@1.20.9 --component calico@3.19.1 --component kvm-operator@update_calico --overwrite
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v15.0.0
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.7.1
```
Comment (ubergesundheit) [inline kvm/v15.0.0/release.yaml:11]: Can we use https://github.com/giantswarm/cert-exporter/releases/tag/v1.8.0 ? Or should I go the usual requests.yaml route?

File: kvm/v15.0.0/release.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# Generated with:
+# devctl release create --name v15.0.0 --provider kvm --base 14.1.2 --app cert-exporter@1.7.1 --app chart-operator@2.18.0 --app coredns@1.6.0@1.8.3 --app net-exporter@1.10.2 --component kubernetes@1.20.9 --component calico@3.19.1 --component kvm-operator@update_calico --overwrite
```
Comment (yulianedyalkova) [inline kvm/v15.0.0/release.yaml:2]: Comment is outdated.

**Review by yulianedyalkova — APPROVED** [review]
There is an outdated comment and kvm-operator is still pointing to a dev version but lgtm otherwise 👍

**Comment by tfussell** [comment]
I will merge this after releasing kvm-operator and fixing the config issue for app-operator 5.2.0.

**Comment by rossf7** [comment]
@tfussell The app-operator config problem needs a fix in cluster-operator. 

We should be using the app-operator-konfigure configmap this change was made in master https://github.com/giantswarm/cluster-operator/pull/1433 but not in the legacy branch :(

Doing that now.

File: kvm/v15.0.0/release.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    releaseOperatorDeploy: false
+    version: 5.2.0
+  - name: calico
+    version: 3.19.2
+  - name: cert-operator
+    releaseOperatorDeploy: true
+    version: 1.0.1
+  - name: cluster-operator
+    releaseOperatorDeploy: true
+    version: 0.27.1
```
Comment (rossf7) [inline kvm/v15.0.0/release.yaml:37]:
```suggestion
    version: 0.27.2
```

@tfussell This has the fix for the app-operator config problem.

Comment (tfussell) [reply kvm/v15.0.0/release.yaml:37]: Thanks for the quick turnaround. I'll retest shortly.

**Pull request merged**